### PR TITLE
[ERMP] Components part 1

### PR DESCRIPTION
**Pull request by AnnZanevskaya**
[Task description](https://epam.sharepoint.com/sites/LMSO/Program/37045f2517-9650-4cd3-b765-53cd8493341a/React%20%E2%80%93%20home%20task.pdf)

- Creat markup for homepage (of [InVisionprototype](https://projects.invisionapp.com/share/F9VXQ7IMZGY/#/screens/406802250) Markup is done with React Components and React Fragments (parent-child).
- Apply necessary styles
- Implement PropTypes for MoveCard component
- Implement ErrorBoundary component for MovieList component

Screenshot: 
image

### CODE REVIEW

**Review by CharlotteAins — APPROVED** [review]
All necessary requirements completed

File: package.json [hunk 1]
```diff
@@ -4,11 +4,18 @@
   "description": "",
   "main": "index.js",
   "dependencies": {
+    "@babel/preset-react": "^7.10.4",
```
Comment (NikitaSedyx) [inline package.json:7]: please try to not mix dev and non dev deps

File: src/Components/AddMovie/index.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const AddMovie = () => (
```
Comment (NikitaSedyx) [inline src/Components/AddMovie/index.jsx:3]: don't create login in `index` files

File: src/Components/Content/index.jsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Wrapper from "../Wrapper";
+import Filter from "../Filter";
+import Sort from "../Sort";
+import MovieList from "../MovieList";
+import ErrorBoundary from "../ErrorBoundary";
+
+const Content = () => (
```
Comment (NikitaSedyx) [inline src/Components/Content/index.jsx:8]: what content? name doesn't say a lo

Comment (NikitaSedyx) [reply src/Components/Content/index.jsx:8]: just imagine I have 100+ pages, how many contents can I have?

File: src/Components/Filter/index.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+            <li className="filter__item filter__item_active">all</li>
+            <li className="filter__item">documentary</li>
+            <li className="filter__item">comedy</li>
+            <li className="filter__item">horror</li>
+            <li className="filter__item">crime</li>
+        </ul>
+    </div>
+)
+
+export default Filter;
```
Comment (NikitaSedyx) [inline src/Components/Filter/index.jsx:15]: EOF

File: src/Components/Header/index.jsx [hunk 5]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Wrapper from "../Wrapper";
+import Logo from "../Logo";
+import AddMovie from "../AddMovie";
+import Search from "../Search";
+
+
```
Comment (NikitaSedyx) [inline src/Components/Header/index.jsx:7]: extra line

File: src/Components/Header/index.jsx [hunk 6]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import Wrapper from "../Wrapper";
```
Comment (NikitaSedyx) [inline src/Components/Header/index.jsx:2]: there is a practice to separate 3rd party imports from source code

File: src/Components/MovieCard/index.jsx [hunk 7]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from 'prop-types';
+
+const MovieCard = (props) => (
```
Comment (NikitaSedyx) [inline src/Components/MovieCard/index.jsx:4]: why not with destrucuring?

File: src/Components/MovieList/index.jsx [hunk 8]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ResultsCount from "../ResultsCount";
+import MovieCard from "../MovieCard";
+import { movies } from "../../Constants/constants.js";
+
+const MovieList = () => (
+    <>
+        <ResultsCount
```
Comment (NikitaSedyx) [inline src/Components/MovieList/index.jsx:8]: smh wrong with codestyle

File: src/Components/MovieList/index.jsx [hunk 9]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import ResultsCount from "../ResultsCount";
+import MovieCard from "../MovieCard";
+import { movies } from "../../Constants/constants.js";
+
+const MovieList = () => (
```
Comment (NikitaSedyx) [inline src/Components/MovieList/index.jsx:6]: would be nice to see movies are passed via props

File: src/Components/ResultsCount/index.jsx [hunk 10]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import PropTypes from 'prop-types';
+
+const ResultsCount = (props) => (
+    <h4 className="results__description"><span className="results__count">{props.count}</span> movies found</h4>
+);
+
+export default ResultsCount;
```
Comment (NikitaSedyx) [inline src/Components/ResultsCount/index.jsx:8]: export is before static props, not the best choice for readability

File: src/Styles/sass/base/_button.scss [hunk 11]
```diff
@@ -0,0 +1,41 @@
+.header__button {
+    margin-top: 10px;
+}
+
+.button {
```
Comment (NikitaSedyx) [inline src/Styles/sass/base/_button.scss:5]: you have everything to use css modules

**Pull request merged**